src/components/action-popover/action-popover.style.js

```diff
@@ -78,7 +75,7 @@ const MenuButton = styled.div`
     cursor: pointer;
   }
   width: fit-content;
-  margin: auto;
+  /* margin: auto; */
```

Comment:
Is it this commented out piece causing the styling bug I commented about here?
https://github.com/Sage/carbon/pull/3508#issuecomment-756064206